tests/fixture/pageObject/Checkout.ts

```diff
@@ -13,28 +13,29 @@ export class Checkout {
         this.playwright = new PlaywrightHelper(page);
     }
 
-    // This is for a developer to debug PollyJS replay quickly.
-    // Should not be used in a final test file.
+    // CheckoutFixtures helper, only used in ../CheckoutFixtures.ts
+    async close(): Promise<void> {
+        await this.playwright.stopAll();
+    }
+
+    // Dev helper
     log(): void {
         this.playwright.enableDevMode();
     }
 
-    async close(): Promise<void> {
-        await this.playwright.stopAll();
+    // Testing environment setup helpers
+    async use(preset: CheckoutPagePreset): Promise<void> {
+        await this.playwright.usePreset(preset);
     }
 
-    async create(har: string, storeUrl: string): Promise<void> {
-        await this.playwright.createCheckout(har, storeUrl);
+    async start(HAR: string): Promise<void> {
```

Comment:
Rename an essential step from `checkout.create()` to `checkout.start()` as we only need a HAR file name now, `start` also sounds more mandatory. We do `checkout.close()` automatically for all tests.